notebooks/cba_introduction.ipynb

```diff
@@ -0,0 +1,1091 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "shp_matrix = cba.add_layer(\n",
```

Comment:
I got an error when running the notebook here:

~~~
AttributeError: 'DataFrame' object has no attribute 'dissolve'
~~~

This is due to the `grid` object being a `DataFrame`. It has a `geom` column but has not been for some reason not been converted to `GeoDataFrame`.

Reply:
Never had this error on my side. I hope the latest modification will remove it, there shouldn't be any conflict between a `geom` and a `geometry` column right now.